components/HeaderConditionSection/index.js

```diff
@@ -0,0 +1,26 @@
+const HeaderConditionSection = () => {
+  return (
+    <div className="flex flex-col md:flex-row md:items-end justify-between p-20">
+      <div className="md:px-8">
+        <h3 className="text-lg md:text-xl font-extrabold">All Categories</h3>
+        <p className="font-light md:font-normal text-sm md:text-base text-gray-500 mb-2">
+          View all of our categories to get the most relevant information for
+          yourself
+        </p>
+      </div>
+      <div className="md:px-10">
+        <label className="text-gray-700 font-light md:font-light text-xs md:text-sm ">
```

Comment:
It's good that you're using a label here Negin. Can you remember how to associate it with the input? I'll give you a hint, it requires another attribute in the label tag (and also one more in the input tag).

If you can get them associated it's better for accessibility. Then, for example, if a user clicks on the label, the browser will place the cursor in the input box for them.

Reply:
Hi @ARTurnerGit, I don't think I have ever done that before. Great to know about it. Thank you